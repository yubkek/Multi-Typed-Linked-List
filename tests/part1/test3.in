NEW 3
1
c
32e2
REMOVE 0
NEW 3
1
2
3
  NEW WHAT
VIEW ALL
NEW 2
{{}
lol

