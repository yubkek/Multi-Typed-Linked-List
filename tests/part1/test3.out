List 0: 1 -> c -> 3200.00
List 0 has been removed.

Number of lists: 0
List 1: 1 -> 2 -> 3
INVALID COMMAND: INPUT
Number of lists: 1
List 1
INVALID COMMAND: NEW